ScattReport.java

```diff
@@ -4,7 +4,7 @@
  * @author Kara Beason
  * @author Cydney Caldwell
  * @author Michelle Melton
- * @version Feb 2017
+ * @version Mar 2017
```

Comment:
You should create a new ScattReport.java file - not rename the Scatt.java file, since that is our main class file. For simplicity, you might just name it Report, since it is in the Scatt project.